source/PlayState.hx

```diff
@@ -930,22 +993,55 @@ class PlayState extends MusicBeatState
 		super.create();
 	}
 
-	function ughIntro()
+	function playCutscene(name:String, atEndOfSong:Bool = false)
 	{
+		#if hxCodec
```

Comment:
same here: #if hxcodec

Reply:
also make two new folders, scripts and shaders inside of assets.

add a readme or something inside of it doesnt really matter